src/widgets/index/__tests__/index-test.ts

```diff
@@ -896,6 +902,132 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/index/js/"
         expect(widget.render).toHaveBeenCalledTimes(0);
       });
     });
+
+    it('calls `render` with `scopedResults` coming from siblings and children', async () => {
+      /* eslint-disable @typescript-eslint/camelcase */
+      const level0 = index({ indexName: 'level0_index_name' });
+      const level1 = index({ indexName: 'level1_index_name' });
+      const level2 = index({ indexName: 'level2_index_name' });
+      const level2_1 = index({ indexName: 'level2_1_index_name' });
```

Comment:
We don't cover by an assertions the use case with siblings. We could add an assertion for the `level2_1`.